src/test/java/duke/task/TaskCreatorTest.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskCreatorTest {
+
+    @Test
+    public void createTodoTest() {
+        assertEquals(TaskCreator.createTask("todo read book").toString(),
+                "[T][ ] read book");
+    }
+
+    @Test
+    public void createDeadlineTest() {
+        assertEquals(TaskCreator.createTask("deadline return book /by 2022-5-12 1800").toString(),
+                "[D][ ] return book (by: May 12 2022 6:00 PM)");
```

Comment:
love the line break after comma, as well as the 8 spaces before the second line :))